plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -423,35 +423,39 @@ function ListMenuItem:update()
 
             local fontsize_info = _fontSize(14, 18)
 
-            local wfileinfo = TextWidget:new{
-                text = fileinfo_str,
-                face = Font:getFace("cfont", fontsize_info),
-                fgcolor = self.file_deleted and Blitbuffer.COLOR_DARK_GRAY or nil,
-            }
-            local wpageinfo = TextWidget:new{
-                text = pages_str,
-                face = Font:getFace("cfont", fontsize_info),
-                fgcolor = self.file_deleted and Blitbuffer.COLOR_DARK_GRAY or nil,
-            }
+            local wright_items = {align = "right"}
+            local wright_right_padding = 0
+            local wright_width = 0
+            local wright
 
-            local wright_width = math.max(wfileinfo:getSize().w, wpageinfo:getSize().w)
-            local wright_right_padding = Screen:scaleBySize(10)
-
-            -- We just built two string to be put one on top of the other, and we want
-            -- the combo centered. Given the nature of our strings (numbers,
-            -- uppercase MB/KB on top text, number and lowercase "page" on bottom text),
-            -- we get the annoying feeling it's not centered but shifted towards top.
-            -- Let's add a small VerticalSpan at top to give a better visual
-            -- feeling of centering.
-            local wright = CenterContainer:new{
-                dimen = Geom:new{ w = wright_width, h = dimen.h },
-                VerticalGroup:new{
-                    align = "right",
-                    VerticalSpan:new{ width = Screen:scaleBySize(2) },
```

Comment:
You removed my VerticalSpan (without a farewell comment :/). You really didn't get my "annoying feeling it's not centered but shifted towards top" ?
(I just tested with and without on my Kobo, and I somehow don't notice the difference... Dunno if other things had changed - or if I just didn't stare at them enough :)

Reply:
There's a 2 pixels difference when simulating a Kindle Paperwhite (before on the left):
![results](https://github.com/koreader/koreader/assets/5104286/046cce1e-13d3-4bfe-892b-41fe23333fc2)
Measurements:
| | Before | After |
|-|--------|-------|
|1| 36     | 34    |
|2| 31     | 29    |
|3|  4.5   |  6.5  |
|4| 10.5   | 12.5  |
|5| 14.5   | 12.5  |
|6| 19.5   | 17.5  |
|7| 27     | 29    |
|8| 21     | 23    |

IMHO, after is subjectively and objectively better (including simpler code), so I removed the hack.